distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -594,27 +596,41 @@ type Table
     ## Creates a new Table with the specified range of rows from the input
        Table.
 
+       Does not support `While` or `Sample`.
+
```

Comment:
I'm not sure we want to have different doc comments for in-memory vs database. Rather, I'd add a section `? Supported range types` to _both_ doc comments (to keep them in sync) and explain that DB does not support these.

At least for now I think we were trying to keep these comments in-sync, so I'd try to keep that, unless we actively decide that we no longer want that - I'm also fine with that.